modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -0,0 +1,200 @@
+| Target Variable | String | Name of a variable on which the operation's output is placed | None | No 
+| Target Value | String | Expression that is evaluated against the operation's output and the effect of that expression is stored in the target variable | #[payload] | No 
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+| *Type* | Any
+| *Attributes Type* | Any
+|===
+
+=== For Configurations
+
+* <<Config>>  
+
+=== Throws
+
+* APIKIT-ASYNCAPI:ASYNCAPI_KIT_INTERNAL_SERVER_ERROR  
+
+== Sources
+
+[[MessageListener]]
+== Message Listener
+`<apikit-asyncapi:message-listener>`
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use | None | *Yes* 
+| Channel Name | String | Channel name | None | *Yes* 
+| Servers | Array of String | List of servers to listen to | None | No 
+| Output MIME Type | String | MIME type of the payload that this operation outputs | None | No 
+| Output Encoding | String | Encoding of the payload that this operation outputs | None | No  
+| Primary Node Only | Boolean | Specifies if this source must be executed on the primary node when running in Cluster | None | No 
+| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component | None | *Yes* 
+| Redelivery Policy | <<RedeliveryPolicy>> | Policy for processing the redelivery of the same message | None | No
+| Body Content | Any | Content of the message body | #[payload] | No
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+| *Type* | Any
+| *Attributes Type* | AsyncMessageAttributes
+|===
+
+=== For Configurations
+
+* <<Config>>  
+
+== Types
+
+[[KafkaConfig]]
+=== Kafka Config
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Server Key | String | Identifier for the server configuration within the AsyncAPI configuration | None | *Yes*
+| Producer Config Ref | String | Reference to the Kafka producer configuration | None | No
+| Consumer Config Ref | String | Reference to the Kafka consumer configuration | None | No
+|======================
+
+[NOTE]
+The *Server Key* is required to uniquely identify the server configuration. The references for producer and consumer configurations are optional and depend on the specific setup and requirements of the Kafka implementation.
+
+[[AnypointMqConfig]]
+=== Anypoint MQ Config
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Server Key | String | Identifier for the server configuration in AsyncAPI | None | *Yes*
+| Config Ref | String | Reference to the Anypoint MQ configuration details | None | No
+|======================
+
+[NOTE]
+The *Server Key* is required to uniquely identify the server configuration. *Config Ref* is optional and used to specify additional configuration details.
+
```

Comment:
Tables for Repeatable In Memory Stream and Repeatable File Store Stream are missing

Reply:
Added both tables. Thanks!